src/common.c

```diff
@@ -1154,6 +1154,28 @@ ofi_addr_list_entry_comp_speed(struct slist_entry *cur, const void *insert)
 	return (cur_addr->speed < insert_addr->speed);
 }
 
+void ofi_chosen_ipaddr(const struct fi_provider *prov,
+			 struct ofi_addr_list_entry *addr_entry)
```

Comment:
addr_list_entry contains an ipstr member.  Can the caller just call FI_INFO directly to print that value?

Reply:
@shefty Yes, it can. Changed.
Does the sockets provider require this approach?

Reply:
We're trying not to change the sockets provider at this point, unless it's critical.  We want to move completely to tcp.

Reply:
@shefty Ok. Would you like to see the ofi_chosen_ipaddr on other ones?

Reply:
No.  I don't see that we need this function at all. It's now a 1-line print statement.

Reply:
@shefty Done.